frontend/tailwind.config.js

```diff
@@ -1,11 +1,14 @@
 // TODO: Use theme config over defining custom styles as much as possible
+const defaultTheme = require('tailwindcss/defaultTheme')
```

Comment:
I moved boxShadow under extend so we don't lose the default classes -- so we can avoid importing the defaultTheme

Reply:
Ok, removed the default theme, I guess you'll move the boxShadow in another commit?

Reply:
yessir!